src/Client/discovery.ts

```diff
@@ -148,22 +150,44 @@ export const determineBestNode = (
   };
 };
 
-function createDeadline(seconds: number) {
+function createGossipCallOptions(
```

Comment:
We could probably use `callArguments` to replace this, and keep the logic in one place.
https://github.com/EventStore/EventStore-Client-NodeJS/blob/3f942549e57ed5424642cec4dc0974069983c904/src/Client/index.ts#L580C13-L580C26

Reply:
Yeah I thought about it but it was breaking more than what I needed. I'm fine doing the change though.